roles/v2/pool/src/lib/mining_pool/message_handler.rs

```diff
@@ -8,23 +8,10 @@ use roles_logic_sv2::{
     parsers::Mining,
     routing_logic::NoRouting,
     selectors::NullDownstreamMiningSelector,
-    utils::Mutex,
+    utils::{hash_rate_to_target, Mutex},
 };
 use std::{convert::TryInto, sync::Arc};
-use tracing::trace;
-// [h/s] Expected hash rate of the device (or cumulative hashrate on the
-// channel if multiple devices are connected downstream) in h/s.
-// Depending on server’s target setting policy, this value can be used for
-// setting a reasonable target for the channel. Proxy MUST send 0.0f when
-// there are no mining devices connected yet.
-pub fn hash_rate_to_target(_hs: f32) -> U256<'static> {
```

Comment:
finally :+1: . This could brake POC2 (pool <-> proxy <-> sv2 cpu miner). Btw if that is the case this should be fixed in my next PR so I think that is fine